src/vm/wren_compiler.c

```diff
@@ -3215,6 +3234,9 @@ static void classDefinition(Compiler* compiler, bool isForeign)
   classVariable.scope = compiler->scopeDepth == -1 ? SCOPE_MODULE : SCOPE_LOCAL;
   classVariable.index = declareNamedVariable(compiler);
   
+  ObjString* className = AS_STRING(wrenNewString(compiler->parser->vm, 
+    compiler->parser->previous.start, compiler->parser->previous.length));
```

Comment:
We create a string for the class name a few lines down in the call to `emitConstant()`, so we should just reuse that object. We can probably just store another pointer straight to it in ClassCompiler, though it feels a little iffy to me to have a pointer to a GC object that we don't trace. It will do the right thing since there will always be another pointer to the same string coming from the constant table, but it feels weird. It's probably enough to just leave a comment on className explaining this.